src/styles/BooksList.css

```diff
@@ -0,0 +1,80 @@
+.navBar {
+  width: 75rem;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+}
+
+.books-section {
+  width: 87.5rem;
+  margin: 0 auto;
+  border-radius: 4px;
+  background-color: #fafafa;
+}
+
+#main-logo {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.books {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.categoryDropdown {
+  width: 6.563rem;
+  height: 1rem;
+  margin: 1.125rem 41.5rem 0.688rem 2.563rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.logoSec {
+  width: 2.813rem;
+  height: 2.813rem;
+  margin: 0 0 0 41.5rem;
+  padding: 0.875rem;
+  border: 1px solid #e8e8e8;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+}
+
+.logo_1 {
+  width: 2.5rem;
+  height: 2.5rem;
+}
+
+.books-table {
+  padding-bottom: 2.5rem;
+  background-color: #fafafa;
+}
```

Comment:
- [ ] <b>REQUIRED:</b> There are still few things we need to fix to make our app look more like the provided design. Kindly review the following issues and make sure to fix them.

| Original      | Yours |
  | ----------- | ----------- |
| ![80047697-0c3f6280-8506-11ea-9416-a4f1800e2329](https://user-images.githubusercontent.com/45217364/130871998-88863e2f-c823-4b7b-a5c4-20162973ff21.png) | ![scrnli_10_3_2021_5-21-55 AM](https://user-images.githubusercontent.com/45217364/135738256-1562aceb-35c6-4d3d-81e8-a35011258d9a.png) |
   - The title must be in one line as it is in the mockup design. Kindly fix it.
   - Category selector must show "categories" as default like how it looks on the mockup design and is should not show the borders.
   - The Y(top, bottom) padding of the book item is way too big. it would be nice to reduce it a little just to make it look more like the mockup design.
   -  The color of your progress bar doesn't match the one in the mockup design. you can use the [colorPicker](https://chrome.google.com/webstore/detail/colorpick-eyedropper/ohcpnigalekghcmgcdcenkpelffpdolg) tool to pick up the exact color from the mockup.
   - There must be a line separating the book list and form.
   - All form elements must have the same height.

Reply:
Done